<?xml version="1.0" ?>
<sdf version="1.8">
  <model name="fendt" canonical_link="base">
    <static> true </static>
    <link name="base">
      <pose relative_to="__model__"> 0 0 0 0 0 0 </pose>
      <visual name="fendt_visual">
        <laser_retro>255</laser_retro>
        <geometry>
         <mesh>
            <uri>models://meshes/fendt.dae</uri>
          <scale>1 1 1</scale>
         </mesh>
       </geometry>
        <material>
          <ambient>0.1 0.1 0.1</ambient>
          <diffuse>0.1 0.1 0.1</diffuse>
          <specular>0.1 0.1 0.1</specular>
        </material>
      </visual>
    </link>
    <plugin
      filename="librandom_mover.so"
      name="lidar_sim::random_mover">
      <move_group>target</move_group>
    </plugin>
  </model>
</sdf>

